### ISSUE TITLE
Text-only-mode for parsing

### ISSUE THREAD

**Issue by evs-chris** [ractivejs/ractive#2028]
Would there be any interest in a flag for text-only parsing? Basically, it would be a flag that turned off readElement (and probably readHtmlComment) in the parser. I've found that this would come in handy for support modules around a Ractive app, like building pages in a CMS or exporting data using a template that may have `<, >, &` in it. Reversing a parse has been mentioned before (#1998), but I don't think that that would entirely suit these use-cases, unless...

If there were a method to reverse a parse that included a flag to only do so partially, this would be covered. Something like `Ractive.unparse(someParsedTemplate, { only: 'elements' })` would work, but looks a bit nasty from where I sit. Comments and alternate suggestions are certainly welcome!

For reference, my primary use-case is a CMS that uses Ractive to assemble bits of Ractive template into whole pages. I'm lazy and didn't want to write my own parser or require users to understand the differences between template flavors, so I have Ractive munge the templates one time through using different delimiters and spit out the result with `.toHTML()` for use on the actual page. This works great except for a few corner cases, like `<` and `>` in expressions and elements with directives.

TL;DR double parsing can lead to tears and dirty [regex](https://xkcd.com/1171/) hacks

**Comment by evs-chris** [comment]
Turns out my brain wasn't fully enabled when I was thinking that an `unparse` would work for expressions that won't parse unless they're in a current mustache.

I was also thinking about ways to implement progressive enhancement, and it occurred to me that skipping the element parse would be good for the server-side render. I haven't done any sort of benchmarking, but it seems reasonable that not parsing would be faster than parsing and unparsing to get directives and whatnot back.
